apps/chat/src/components/Markdown/ChatMDComponent.tsx

```diff
@@ -110,7 +110,7 @@ const ChatMDComponent = ({
   const isOverlay = useAppSelector(SettingsSelectors.selectIsOverlay);
 
   const mdClassNames = classnames(
-    'prose min-w-full dark:prose-invert prose-a:text-accent-primary prose-a:no-underline hover:prose-a:underline',
+    'prose min-w-full whitespace-pre-wrap dark:prose-invert prose-a:text-accent-primary prose-a:no-underline hover:prose-a:underline',
```

Comment:
What we need here is an investagation. We can't just add this style. Models can hallucinate. This can be easily reproduced by asking the model to give a nested list (e.g. with three levels of nesting).